lib/common/subredditPost.dart

```diff
@@ -26,6 +26,13 @@ class SubredditPost extends StatelessWidget {
     this.onCommentTap,
     this.selfText,
   });
+  @override
+  _SubredditPostState createState() => _SubredditPostState();
+}
+
+class _SubredditPostState extends State<SubredditPost> {
+  bool upvoted = false;
```

Comment:
Could we make this an enum instead of tracking two bool values?

```
enum SubredditVoteStatus {
      upvoted,
      downvoted,
      neutral
}
```
You can change the wording, but something like that. That way we only have to track one variable instead of having to enable / disable two different bools.

EDIT: Looks there is already a bool defined in the API we use for reddit

```
enum VoteState {
  none,
  upvoted,
  downvoted,
}
```

use this

Reply:
Hi, thank you for your suggestion, I changed the code and now I have only a single variable to track the status of the value.